index_ivf.go

```diff
@@ -37,3 +38,13 @@ func (idx *IndexImpl) GetSubIndex() (*IndexImpl, error) {
 
 	return &IndexImpl{&faissIndex{subIdx}}, nil
 }
+
+// pass nprobe to be set as index time option.
+// varying nprobe impacts recall but with an increase in latency.
+func (idx *IndexImpl) SetNProbe(nprobe int32) {
+	ivfPtr := C.faiss_IndexIVF_cast(idx.cPtr())
+	if ivfPtr == nil {
+		return
+	}
+	C.faiss_IndexIVF_set_nprobe(ivfPtr, C.ulong(nprobe))
```

Comment:
Also, where's the PR that introduces this method. If you've included it within faiss_c.h (reason why you're importing it) - then it's not the right place. IndexIVF_c_ex.h should be the place.

Reply:
I didn't require a FAISS PR since I use an existing setter function - https://github.com/blevesearch/faiss/blob/bleve/c_api/IndexIVF_c.h#L62